modules/lib.lua

```diff
@@ -90,6 +173,10 @@ end
 ---@param entity integer
 ---@return 'North' | 'South' | 'East' | 'West'
 function qbx.getCardinalDirection(entity)
+    if not DoesEntityExist(entity) then
+        return lib.print.error('Cardinal Direction Error')
+    end
+
```

Comment:
I'm not sure whether I want this added directly to the lib or just handled by utils & the QB bridge. I kinda didn't want to handle user errors in the lib, much less like this because it adds a weird hardcoded return value.

Reply:
lib.print.error doesn't return anything either so, this is still correct

Reply:
It isn't correct both in the sense that the function would return `nil` when I don't want the function to handle that, and also in the sense that this makes the utils & the QB bridge not actually handle this the same way QB does (which is why this was added in the first place).

Reply:
Either we change it here to just return the string or we change the bridge to return that when it returns nil

Reply:
I want this to not be handled by the lib at all, and instead to have utils and the QB bridge wrap this function and check for the entity beforehand.

Other string and vehicle related functions had this type of behavior removed so I don't want this  added here either.

Reply:
The utils will be removed soon, so I only added it to the bridge

Reply:
I forgot that I already did the utils part, so yeah only the bridge was needed.